terminus-ui/src/icon/icon.component.ts

```diff
@@ -11,13 +11,17 @@ import { coerceBooleanProperty } from '@terminus/ngx-tools/coercion';
 
 import { TsStyleThemeTypes } from './../utilities/types/style-theme.types';
 import { CSV_ICON } from './custom-icons/csv';
+import { ENGAGE_ICON } from './custom-icons/engage';
+import { LIGHTBULB_ICON } from './custom-icons/lightbulb';
 
 
 /**
  * Currently supported custom icons
  */
 export type TS_CUSTOM_ICON
-  = 'csv'
+  = 'csv'     |
+    'engage'  |
+    'lightbulb'
 ;
```

Comment:
Looking through the library it seems our type formatting varies a bit. My vote is to align our formatting to match the button type:

https://github.com/GetTerminus/terminus-ui/blob/ef1ec246451b313a1c225753bfc41ebe7f844e8a/terminus-ui/src/button/button.component.ts#L28-L33